pkg/api/rebalance_req.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import "github.com/pborman/uuid"
+
+// RebalanceStatus represents Rebalance Status
+type RebalanceStatus uint64
+
+const (
+	// Started should be set only for a volume that has been just started rebalance process
+	Started RebalanceStatus = iota
+	// Inprogress should be set only for a volume that are running rebalance process
+	Inprogress
+	// Failed should be set only for a volume that are failed to run rebalance process
+	Failed
+	// Completed sholud be set only for a volume that the rebalance process is completed
```

Comment:
Each of these are specific to the rebalance process on a node. So a volume with 3 nodes can have one rebalance process that has failed, one in progress and one that has completed.